documentation/modules/ROOT/pages/connectors/mysql.adoc

```diff
@@ -1024,6 +1025,19 @@ MySQL connector events are designed to work with link:{link-kafka-docs}/#compact
 === Tombstone events
 When a row is deleted, the _delete_ event value still works with log compaction, because Kafka can remove all earlier messages that have that same key. However, for Kafka to remove all messages that have that same key, the message value must be `null`. To make this possible, after {prodname}’s MySQL connector emits a _delete_ event, the connector emits a special tombstone event that has the same key but a `null` value.
 
+// Type: continue
+[id="mysql-snapshot-events"]
+=== Snapshot events
+MySql connector events are emitted as `READ` events. In case you want the connector to emit events as `CREATE`, it can be handled using a Simple Message Transforms (SMT).
```

Comment:
```suggestion
The MySql connector emits snapshot events using the "r" operation type (`READ`). In case you want the connector to emit snapshot events as "c" events (`CREATE`, as done incorrectly in earlier versions), this can be achieved using a Simple Message Transforms (SMT).
```

@jpechane, what did we say, do we actually want to mention this _at all_ in the docs?

Reply:
If we do want to mention it, it needs the right conditionals to be excluded from downstream documentation.